package.json

```diff
@@ -29,10 +29,10 @@
   },
   "homepage": "https://github.com/team-767/react-with-hover#readme",
   "dependencies": {
-    "recompose": ">= 0.19.0 < 1.0"
+    "recompose": ">= 0.26.0"
   },
   "peerDependencies": {
-    "react": ">= 0.14"
+    "react": ">= 0.16"
```

Comment:
What do you think of `>= 15.0.0` ? Since after react 15 alpha they bump the major version.

Reply:
React 16 was the first major rewrite of the library and lots of thing have changed. That said I believe that it would still work with react 15 so I'll change it.